Gemfile

```diff
@@ -104,6 +104,13 @@ group :development, :test do
   gem 'pry-awesome_print'
   gem 'pry-byebug'
   gem 'rspec-rails'
+
+  # Profiling for use in prod
+  gem 'flamegraph'
+  gem 'memory_profiler'
+  gem 'meta_request'
+  gem 'rack-mini-profiler', require: false
+  gem 'stackprof'
```

Comment:
These are dev tools so I moved it here. We can move it back in case it is being used in prod>

Reply:
We absolutely do use rack-mini-profiler in production. I'm not sure about the other ones, however.